src/neo/Oracle/OracleResultError.cs

```diff
@@ -0,0 +1,35 @@
+namespace Neo.Oracle
+{
+    public enum OracleResultError : byte
+    {
+        /// <summary>
+        /// There was no errors
+        /// </summary>
+        None = 0x00,
+
+        /// <summary>
+        /// Timeout
+        /// </summary>
+        Timeout = 0x01,
+
+        /// <summary>
+        /// There was an error with the server
+        /// </summary>
+        ServerError = 0x02,
+
+        /// <summary>
+        /// There was an error with the policy
+        /// </summary>
+        PolicyError = 0x03,
```

Comment:
`PolicyError` may include not enough fee for the service requested?

Reply:
I think that maybe in the future oracle could ban by policy specific hosts.